web/packages/teleport/src/components/TdpClientCanvas/TdpClientCanvas.tsx

```diff
@@ -97,6 +97,47 @@ function TdpClientCanvas(props: Props) {
     }
   }, [client, clientOnPngFrame]);
 
+  let previousCursor = 'auto';
+
+  useEffect(() => {
+    if (client) {
+      const canvas = canvasRef.current;
+      const updatePointer = (pointer: {
+        data: ImageData | boolean;
+        hotspot_x?: number;
+        hotspot_y?: number;
+      }) => {
+        if (typeof pointer.data === 'boolean') {
+          if (pointer.data) {
+            canvas.style.cursor = previousCursor;
+          } else {
+            previousCursor = canvas.style.cursor;
+            canvas.style.cursor = 'none';
+          }
+          return;
+        }
+        const cursor = document.createElement('canvas');
+        cursor.width = pointer.data.width;
+        cursor.height = pointer.data.height;
+        cursor.getContext('2d').putImageData(pointer.data, 0, 0);
+        canvas.style.cursor =
+          'url(' +
+          cursor.toDataURL() +
+          ') ' +
+          pointer.hotspot_x +
+          ' ' +
+          pointer.hotspot_y +
+          ', auto';
```

Comment:
I think you can rewrite this with string interpoloation:

```
canvas.style.cursor = `url('${cursor.toDataURL()') ${pointer.hotspot_x} ${pointer.hotspot_y} auto`
```

Reply:
Yep, you can, changed